bindgen/clang.rs

```diff
@@ -1923,9 +1923,9 @@ pub(crate) struct UnsavedFile {
 
 impl UnsavedFile {
     /// Construct a new unsaved file with the given `name` and `contents`.
-    pub(crate) fn new(name: String, contents: String) -> UnsavedFile {
-        let name = CString::new(name).unwrap();
-        let contents = CString::new(contents).unwrap();
+    pub(crate) fn new(name: Box<str>, contents: Box<str>) -> UnsavedFile {
```

Comment:
I would use `&str` here, as it is less unusual... just checked locally, and the only other change needed is the caller args.